src/shared-components/dropdown/index.tsx

```diff
@@ -112,17 +114,17 @@ export const Dropdown = ({
 
 Dropdown.propTypes = {
   borderRadius: PropTypes.string,
-  value: allowNullPropType(
-    PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  ),
+  onChange: PropTypes.func.isRequired,
   options: PropTypes.arrayOf(
     PropTypes.shape({
       value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
       label: PropTypes.string,
       disabled: PropTypes.bool,
     }),
   ).isRequired,
-  textAlign: PropTypes.oneOf(['left', 'center']),
-  onChange: PropTypes.func.isRequired,
   optionsContainerMaxHeight: PropTypes.string,
+  textAlign: PropTypes.oneOf(['left', 'center']),
+  value: allowNullPropType(
```

Comment:
I dont get this, value was never set as default null so the only reason we should leave this here is if we want consumers to pass null to value explicitly

Reply:
If we tighten up the types / defaults, I'm comfortable removing this utility from `radiance-ui`.